packages/connectivity/lib/connectivity.dart

```diff
@@ -60,11 +81,24 @@ class Connectivity {
   /// make a network request. It only gives you the radio status.
   ///
   /// Instead listen for connectivity changes via [onConnectivityChanged] stream.
-  Future<ConnectivityResult> checkConnectivity() async {
+  Future<ConnectivityResult> checkConnectivity(
```

Comment:
`checkSubtype` seems not used. Also add some doc to link to the [getNetworkSubtype] for people who wants to check the subtype.

Reply:
i added some docs. Please review and let me know if they are up to standard.